Screenbox/App.xaml

```diff
@@ -17,12 +17,49 @@
                     <ResourceDictionary.ThemeDictionaries>
                         <ResourceDictionary x:Key="Default">
                             <StaticResource x:Key="AccentFillColorCustomBrush" ResourceKey="SystemAccentColorLight2" />
+
+                            <!--  Caption button  -->
+                            <StaticResource x:Key="CaptionButtonBackground" ResourceKey="SubtleFillColorTransparent" />
+                            <StaticResource x:Key="CaptionButtonBackgroundPointerOver" ResourceKey="SubtleFillColorSecondary" />
+                            <StaticResource x:Key="CaptionButtonBackgroundPressed" ResourceKey="SubtleFillColorTertiary" />
+                            <!--  Foreground colors ignore the alpha channel, so we flattened them by layering them over the SolidBackgroundFillColorBase and its associated subtle color  -->
+                            <!--  TextFillColorPrimary  -->
+                            <Color x:Key="CaptionButtonForeground">#FFFFFF</Color>
+                            <!--  TextFillColorPrimary  -->
+                            <Color x:Key="CaptionButtonForegroundPointerOver">#FFFFFF</Color>
+                            <!--  TextFillColorSecondary  -->
+                            <Color x:Key="CaptionButtonForegroundPressed">#D1D1D1</Color>
+                            <!--  TextFillColorDisabled  -->
+                            <Color x:Key="CaptionButtonForegroundInactive">#717171</Color>
```

Comment:
There is no way to explicitly get a theme value, so you may need to convert these into static resources with names specific to the Dark and Light themes.